tensorflow/lite/micro/examples/hello_world/evaluate_test.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import numpy as np
+
+from tensorflow.python.framework import test_util
+from tensorflow.python.platform import resource_loader
+from tensorflow.python.platform import test
+from tflite_micro.tensorflow.lite.micro.python.interpreter.src import tflm_runtime
+from tflite_micro.tensorflow.lite.micro.examples.hello_world import evaluate
+
+PREFIX_PATH = resource_loader.get_path_to_datafile('')
+
```

Comment:
leading underscore for internal module level constants

`_PREFIX_PATH`

https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations

Reply:
Done.